testnet/src/event_dispatcher.rs

```diff
@@ -86,11 +85,40 @@ impl EventObserver {
         }).to_string();
 
         // Send payload
-        let res = stream.write_bufs(&vec![payload.as_bytes().into()]);
-        if let Err(err) = res {
-            error!("Event dispatcher failed sending buffer: {:?}", err);
-            panic!();
-        }
+        let mut retry_count = 0;
+        let mut backoff: f64 = 1.0;
+        let mut rng = thread_rng();
+        loop {
+            let response = Client::new()
+                .post(&self.endpoint)
+                .json(&payload)
+                .send();
+
+            match response {
+                Ok(response) => {
+                    if response.status() == StatusCode::OK {
```

Comment:
This should check the status code with `.status().is_success()` just in case a sidecar implementation ever wants to send other 2xx status codes (e.g., 202).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1535/commits/851bec83c3cbae8515867f260e109da6f4c744d7